src/components/FeatureViews/Search/Search.js

```diff
@@ -82,9 +85,21 @@ const Search = (props) => {
             if(sortAccount){
                 baseConfig.params["sort"] = sortAccount;
             }
-            setAccountRequestConfig(baseConfig);
-            api.updateSearchStatus(NOT_SEARCHING);
-            filterSuggestions();
+
+            let toUpdate = false;
+            setAccountRequestConfig(() => {
+                if(JSON.stringify(accountRequestConfig) !== JSON.stringify(baseConfig)){
+                    toUpdate = true;
+                    return baseConfig
+                }
+            });
```

Comment:
Can this be replaced with:

    toUpdate = /* if condition */
    setAccountRequestConfig(baseConfig);

?